### ISSUE TITLE
EXASOL: L013 wrong result when using json_extract

### ISSUE THREAD

**Issue by sti0** [sqlfluff/sqlfluff#2361]
In Exasol we could use the `json_extract()` function to extract multiple values out of a json string. The `emits` clause defines how the values are exposed. In such a case its not possible to use a alias.

## Expected Behaviour
L013 should not raise when using `json_extract()`

## Observed Behaviour
L013 raises.

## Steps to Reproduce
```sql
select
    somecol,
    json_extract(json_str, '$.AFIELD', '$.BFIELD') emits (cola char(1), colb char(1))
from abc;
```

## Dialect
Exasol

## Version
Include the output of `sqlfluff --version` along with your Python version.
If you are using dbt then please additionally include the dbt version.

sqlfluff v0.9.1

## Configuration
```
Include your SQLFluff configuration here
```
